SECURITY.MD

```diff
@@ -0,0 +1,19 @@
+# O3DE Security Policy
+
+## Reporting a Vulnerability
+
+If you have information about a security issue or vulnerability in O3DE, please send an e-mail to [security@o3de.org](mailto:security@o3de.org).
```

Comment:
Can we add a sentence or bullet after this that says (in emphasized text) "Please do not submit a GitHub issue to report a suspected security problem."

Reply:
Have changes to issue reporting here: https://github.com/o3de/o3de/pull/8040.

Reply:
That's good, but I think we need to state that outright here in this document as well.

Reply:
Done